hoomd/md/EvaluatorPairMie.h

```diff
@@ -58,14 +58,16 @@ class EvaluatorPairMie
             Scalar m2;
             Scalar m3;
             Scalar m4;
+	    Scalar mie_n;
```

Comment:
There should be a way to implement this without storing additional parameters. If you transform m and n and when you store them in `m3` and `m4`, then you can apply the reverse transform to the numbers in `asDict`

Reply:
For now I renamed the transformed `n` and `m` to `mie_log2_n` and `mie_log2_m` for clarity.  Since `m3` and `m4` are used to directly compute the prefactor `outFront`, `m1`, and `m2`, I'm not sure if there's a way to store the transformed values without additional parameters.  Perhaps I'm misunderstanding.